### PR TITLE
/download/desktop refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit) and [mock-up](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=1%3A30&mode=dev)
- Drive bys:
  - Renamed some existing partials for increased specificity (eg: /_resources -> /_instructions-resources). 
  - Divided the corresponding thank-you page into partials which this landing page also reuses

## QA

- View the site locally in your web browser at: https://ubuntu-com-13697.demos.haus/download/desktop
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8491

### CODE REVIEW

**Comment by juanruitina** [comment]
So exciting to see this built! A few comments:

- [ ] Mascot!
- [ ] Add LTS to the 22.04.4 CTA, `<abbr title="Long-term support">LTS</abbr>`
- [ ] Please have the size spell units as "GB", "MB"... instead of "G", "M"..., so it's in line with our copy style guide.
- [ ] Remove the automatic scrolling when clicking on the tabs
- [ ] Remove the GNOME line in the System requirements tab
- [ ] Add the "[Follow the step-by-step tutorial ›](https://ubuntu.com/tutorials/install-ubuntu-desktop)" link under the How to install tab
- [ ] Make the two instances of the "For other versions of Ubuntu Desktop including torrents..." paragraph small, and please replace "see" with "check out" (just tweaked it in the copy doc)
- [ ] Please add a bit of spacing above "Help is always at hand" on small and medium viewports
- [ ] Links next to buttons wrap weirdly on smaller viewports: they should move as a whole to a separate line before wrapping within. Maybe flex is the best way to achieve that?

Screenshot 2024-03-27 at 11 34 46
Screenshot 2024-03-27 at 11 34 51

**Comment by mtruj013** [comment]
Thanks @juanruitina, applied most of the changes, but about this one:
> Please add a bit of spacing above "Help is always at hand" on small and medium viewports

This already has padding-top of .4rem applied to it from the `p-muted-heading` class. Increasing that spacing would mean that the spacing above the "Resources" heading no longer matches, which seems rather strange and counterintuitive to feedback we've received before. Regardless, if indeed the top padding is to increase for both of these please let me know the exact value you would like (.5rem nudge?) and I'll implement 

image

cc: @lyubomir-popov

**Comment by juanruitina** [comment]
About the spacing: I'm not sure of how to implement it, but it's problematic to keep that very limited padding when "Resources" and "Help is always at hand" are stacked (i.e., on medium and small). Hierarchy is not clear, see here:

ubuntu-com-13697 demos haus_download_desktop(iPhone 12 Pro)

Even a 1rem spacing in between would help. 

>  Text (links) next to buttons wrap weirdly on smaller viewports: they should move as a whole to a separate line before wrapping within. Maybe flex is the best way to achieve that? Please align with @carkod on a solution, similar issue on his pages.

This is still not fixed, can you check?

Please remove the spaces between file size and unit (4.7GB). I don't love it, but it's [the rule](https://docs.google.com/document/d/1AX-kSNztuAmShEoohe8L3LNLRnSKF7I0qkZGNeoGOok/edit#heading=h.6tod28b1qlgf).

And one visual note: the spacing between H2 and mascot is 64px (the regular one). Not sure how it should be treated on smaller viewports, I leave that to @lyubomir-popov

**Comment by mtruj013** [comment]
@juanruitina applied the fix for the link wrapping on small/medium vps and the white space in the download size , but I'll wait until @lyubomir-popov gets back to us about the "Help is always at hand" spacing on smaller screens (no point in me trying to just guess what would get design +1). Having medium and small screen views included in the mock-up would've been really useful here, but I know there's no design bandwidth for that 😢

**Comment by juanruitina** [comment]
Thank you 🙏 Let me flex my flex knowledge and make a suggestion: I think we can use `flex-direction: row; flex-wrap: wrap;` instead of `flex-direction: column`. That way it only wraps if needed, and you can even makes get rid of the media queries. Also, maybe use `row-gap: .5rem;` instead of just `gap` given the buttons and links already have their spacing.

+1ing this already, just please find some solution for the spacing issue (and thank you for dealing with the responsiveness messiness 🙃)

**Comment by mtruj013** [comment]
> Let me flex my flex knowledge

Flex<sup>2</sup> ! Much better fix indeed, updated 🙂 

cc: @carkod since you were having the same issue

**Comment by petesfrench** [comment]
Couldn't add a note in the diff, but it looks like this link is missing a little arrow thingy:
Screenshot from 2024-03-28 13-42-26

**Comment by lyubomir-popov** [comment]
@mtruj013 looks good, a few small things to fix:

- this heading needs to be inside a p-section to push the mascot down as in the design:
image

- more space here (p-section--shallow) for the content preceding the tabs - so there's more space above the tabs 
image


- no need for the indentation before the certified logo: 
image

- the design doesn't include a vertical divider, but it can remain if someone else insists ^

- the lines here have to go all the way on medium screens. currently there's no line above the icons. Also, I'd do a 50-50 on medium screens too, like so:
image

I'd move the p-section wrapper in one level, so on medium the two sections have 4rem margin-bottom. Currently, the help heading is too close to the 3 columns above it: 
image

 
image

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -1,274 +1,185 @@
...
+        </a>      
       </p>
-      <p>
-        <a href="https://cdimage.ubuntu.com/releases/mantic/release/ubuntu-{{ releases.latest.short_version }}-desktop-legacy-amd64.iso"
-          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ releases.latest.short_version }} legacy installer', 'eventValue' : undefined });">Download
-          {{ releases.latest.short_version }} (Legacy Desktop Installer)</a>
-      </p>
-      <p><small>For other versions of Ubuntu Desktop including torrents, the network installer, a list of local mirrors
-          and past releases <a href="/download/alternative-downloads">see our alternative downloads</a>.</small></p>
+      <p class="p-text--small">For other versions of Ubuntu Desktop including torrents, the network installer, a list of local mirrors and past releases <a href="/download/alternative-downloads">checkout our alternative downloads</a>.</p>
```
Comment (petesfrench) [inline templates/download/desktop/index.html:146]:
```suggestion
      <p class="p-text--small">For other versions of Ubuntu Desktop including torrents, the network installer, a list of local mirrors and past releases <a href="/download/alternative-downloads">check out our alternative downloads</a>.</p>
```

File: templates/download/desktop/index.html [hunk 2]
```diff
@@ -1,274 +1,185 @@
...
-        height="200",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-        }}
-      </div>
-      <p>
-        <a class="p-button--positive is-wide"
+    <div class="col">
+      <p>The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 23.10 comes with nine months of security and maintenance updates, until July 2024. <a href="{{ releases.latest.release_notes_url }}">Ubuntu {{ releases.latest.short_version }} release notes</a></p>
```
Comment (petesfrench) [inline templates/download/desktop/index.html:129]:
```suggestion
      <p>The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu 23.10 comes with nine months of security and maintenance updates, until July 2024. <a href="{{ releases.latest.release_notes_url }}">Ubuntu {{ releases.latest.short_version }} release notes</a>.</p>
```

File: templates/download/shared/_secure-enterprise-management.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Secure enterprise management<br class="u-hide--medium u-hide--small"/> with Ubuntu Pro Desktop</h2>
+    </div>
+    <div class="col">
+      <p>Ubuntu Pro Desktop is a comprehensive subscription delivering enterprise-grade security, management tooling, and extended support for developers and organisations. Ubuntu Pro Desktop is free for personal use on up to five machines.</p>
+      <hr class="p-rule" />
+      <ul class="p-list--divided u-no-padding--bottom">
+        <li class="p-list__item is-ticked">Security updates for the <a href="/esm">full open source stack</a></li>
+        <li class="p-list__item is-ticked">Advanced <a href="/engage/microsoft-active-directory">Active Directory</a> and LDAP integration.</li>
```
Comment (petesfrench) [inline templates/download/shared/_secure-enterprise-management.html:26]:
```suggestion
        <li class="p-list__item is-ticked">Advanced <a href="/engage/microsoft-active-directory">Active Directory</a> and LDAP integration</li>
```

**Comment by petesfrench** [comment]
these images look kinda low res when zoomed in on:
image

**Review by petesfrench — APPROVED** [review]
LGTM, just a couple super minor things

**Comment by juanruitina** [comment]
Fixed #13697

**Pull request merged**